src/oc/interaction/api/comments.clj

```diff
@@ -83,10 +177,19 @@
 (defn routes [sys]
   (let [db-pool (-> sys :db-pool :pool)]
     (compojure/routes
+
       ;; Comment listing and creation
       (ANY "/orgs/:org-uuid/boards/:board-uuid/topics/:topic-slug/entries/:entry-uuid/comments"
         [org-uuid board-uuid topic-slug entry-uuid]
         (pool/with-pool [conn db-pool] (comment-list conn org-uuid board-uuid topic-slug entry-uuid)))
       (ANY "/orgs/:org-uuid/boards/:board-uuid/topics/:topic-slug/entries/:entry-uuid/comments/"
         [org-uuid board-uuid topic-slug entry-uuid]
-        (pool/with-pool [conn db-pool] (comment-list conn org-uuid board-uuid topic-slug entry-uuid))))))
\ No newline at end of file
+        (pool/with-pool [conn db-pool] (comment-list conn org-uuid board-uuid topic-slug entry-uuid)))
+
+      ;; Comment editing and removal
+      (ANY "/orgs/:org-uuid/boards/:board-uuid/topics/:topic-slug/entries/:entry-uuid/comments/:comment-uuid"
+        [org-uuid board-uuid topic-slug entry-uuid comment-uuid]
+        (pool/with-pool [conn db-pool] (comment-item conn org-uuid board-uuid topic-slug entry-uuid comment-uuid)))
+      (ANY "/orgs/:org-uuid/boards/:board-uuid/topics/:topic-slug/entries/:entry-uuid/comments/:comment-uuid/"
```

Comment:
Only difference is the trailing `/`. We have the duplicated routes, with and w/o trailing `/` in most of the HTTP services. Seems to be a limitation of compojure that you can't create one route that accounts for both.